TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -397,6 +397,10 @@ TYPO3:
             authenticationProviderName: '${entity.authenticationProviderName}'
             name: '${entity.party.name.fullName}'
 
+    login:
+      styles:
```

Comment:
I don't have a better suggestion, but I find `TYPO3.Neos.login.styles` a bit confusing for this setting:
"login" is very generic, but it only targets the *Neos backend login form*, right? So maybe call it `backendLoginForm`?
And "stylesheets" would be more in sync with [Neos templating] (http://neos.readthedocs.org/en/latest/CreatingASite/RenderingCustomMarkup/AdjustingOutput.html#the-head) than "styles"